bridge_adaptivity/module/models.py

```diff
@@ -54,7 +54,7 @@ def _load_cls_from_applicable_module(module_path, mod_name, class_startswith=Non
     name: _load_cls_from_applicable_module("module.policies", file_name, class_endswith="GradingPolicy")
     for file_name, name in GRADING_POLICY_MODULES
 }
-GRADING_POLICY_CHOICES = ((k, v.public_name) for k, v in GRADING_POLICY_NAME_TO_CLS.items())
+GRADING_POLICY_CHOICES = ((k, v) for k, v in GRADING_POLICY_NAME_TO_CLS.items())
```

Comment:
Could you please explain why do we change tuple generator public_name strings with the classes instances? For what purpose?